tests/internal/s3.js

```diff
@@ -48,10 +48,10 @@ export function s3TestSuite(data) {
         // Assert
         expect(gotFirstObject).to.be.an('object')
         expect(gotFirstObject.key).to.equal(data.s3.testObjects[0].key)
-        expect(gotFirstObject.body).to.equal(data.s3.testObjects[0].data)
+        expect(gotFirstObject.data).to.equal(data.s3.testObjects[0].body)
```

Comment:
These `getObject` tests were broken, but they were passing since both values were undefined.

Reply:
Good catch indeed 👍🏻 Thanks for fixing it on the way 🙇🏻